docs/current/guides/488564-openshift-gitlab.md

```diff
@@ -0,0 +1,110 @@
+
+  ```shell
+  oc adm taint nodes NODE-NAME builder-node=true:NoSchedule
+  ```
+
+1. Install the Dagger Engine using the Dagger Helm chart:
+
+  ```shell
+  helm upgrade --create-namespace --install --namespace dagger dagger oci://registry.dagger.io/dagger-helm -f values.yaml
+  ```
+
+1. Grant the necessary permissions for the `default` service account in the `dagger` namespace:
+
+  :::info
+  Without this step, pod creation will fail due to insufficient permissions to execute privileged containers with fixed user IDs and host path volume mounts.
+  :::
+
+  ```shell
+  oc adm policy add-scc-to-user privileged -z default -n dagger
+  ```
+
+## Step 2: Configure GitLab Runner
+
+The next step is to configure a GitLab Runner. Follow these steps:
+
+1. Follow the process to [obtain a runner authentication token](https://docs.gitlab.com/ee/ci/runners/runners_scope.html#create-a-shared-runner-with-a-runner-authentication-token). Note that it is not necessary to register the runner from the command line during this process.
+
+1. Create the configuration for the GitLab Runner as `runner-config.yaml` and `runner.yaml`. Replace the `YOUR-GITLAB-URL` placeholder with the URL of your GitLab instance and replace the `YOUR-GITLAB-RUNNER-TOKEN-REFERENCE` placeholder with the token obtained above.
+
+  ```yaml title=runner-config.yaml file=./snippets/openshift-gitlab/runner-config.yaml
+  ```
+
+  ```yaml title=runner.yaml file=./snippets/openshift-gitlab/runner.yaml
+  ```
+
+  This configuration uses a similar configuration as that seen in Step 1 for the taints and tolerations and the pod affinity. This ensures that the GitLab runner worker pods only runs on nodes with Dagger engines.
+
+1. Apply the configuration and deploy the GitLab Runner:
+
+  ```shell
+  oc apply -f runner-config.yaml -n dagger
+  oc apply -f runner.yaml -n dagger
+  ```
+
+## Step 3: Create a GitLab CI/CD pipeline
+
+1. For Dagger Cloud users only, [add a new CI/CD variable in GitLab](https://docs.gitlab.com/ee/ci/variables/#define-a-cicd-variable-in-the-ui) with the name `DAGGER_CLOUD_TOKEN` and set its value to the Dagger Cloud token.
+
+1. Create a new GitLab CI/CD pipeline configuration file in your repository at `.gitlab-ci.yml` with the following content:
+
+  ```yaml title=.gitlab-ci.yml file=./snippets/openshift-gitlab/.gitlab-ci.yml
+  ```
+
+  The most important sections of this configuration are:
+
+  - The `tags` entry, which tells GitLab to use the GitLab Runner which is connected to the Dagger Engine;
+  - The `_EXPERIMENTAL_DAGGER_RUNNER_HOST` variable, which specifies the socket for the Dagger CLI to use.
+
+## Step 4: Run a GitLab CI job
+
+At this point, the deployment is configured and ready for use. Test it by committing a new change to the source code repository, which should trigger the GitLab CI pipeline. Your CI pipelines should now be connected to your Dagger Engines.
```

Comment:
We should add 1-2 sentences here about how the user can inspect the logs...kubectl? Gitlab UI?